test/validation/org/apache/cassandra/HarnessTest.java

```diff
@@ -236,6 +243,7 @@ public FailureTask(String moduleName, String message)
 
         public void run()
         {
+            this.message = this.moduleName + ": " + this.message;
```

Comment:
This isn't what I had in mind, but works just as well and is probably cleaner. Very nice.